FakeItEasy.sln

```diff
@@ -1,26 +1,32 @@
-﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.31101.0

```

Comment:
I'd rather keep this PR focused on thing. I'll push the amend so we can merge this and we'll tackle restricting to VS2015+ separately.